Dockerfile

```
@@ -0,0 +1,33 @@
+# syntax=docker/dockerfile:1
+ARG BUILDER_REPO_PATH=/builder/alicia-server
+
+FROM ubuntu:latest AS builder
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends build-essential git cmake libboost-dev libpq-dev
+RUN apt-get clean
+
+ARG BUILDER_REPO_PATH
+ARG BUILD_TYPE=Release
+
+WORKDIR ${BUILDER_REPO_PATH}
+COPY . .
+RUN git submodule update --init --recursive
+
+RUN cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} . -B ./build
```

We'll never be running tests in the docker container, we don't have to compile them. 
```suggestion
RUN cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DBUILD_TESTS=False . -B ./build
```